src/Client.php

```diff
@@ -6,22 +6,23 @@
 
 class Client
 {
-    public $cuid;
-    
-    public $member;
+    public int $cuid;
+
+    public ?int $member, $group;
     
     const API_URL = 'https://www.autoscout24.ch/api/hci/v3/json/';
     
-    public function __construct($cuid, $member)
+    public function __construct(int $cuid, ?int $member, ?int $group)
```

Comment:
I have added php 7.1 as min requirement (which is introduced by your changes), thats fine.

But i think you want to have the $member and or $group params optional. `?int` will allow null values, but the parameters are still required. So i think what you want it is: 

```
__construct(int $cuid, ?int $member = null, ?int $group = null)
```

And then check in the constructor whether $member or $group is provided?

Or can you tell me more about the group parameter which i am not aware of?

Reply:
It think the behaviour for the developer using this library is the same with both versions. The documentation states that either `$group` or `$member `need to be passed. Making both null by default does not force the developer to actively set one of them. 

In my case, i only need the group parameter. Having `$member `set to null seems fine to me, since the `http_build_query` function automatically removes null parameters.

As far as i can tell from the documentation the group parameters allows querying all vehicles of the "dealerGroup". The scope is requests seem to include multiple "members" added to a dealerGroup.

Reply:
is either `$member` or `$group` required? Then your solution would be find but we should check if both are empty, then throw an exception.

Reply:
Quote from the documentation: 

Each request must include the two parameters: 
- cuid={accountId}
- member={accountId} or group={dealerGroupId}

I omitted the exception because the api will return useless data that could be checked inside the application. I am not sure if it is better to check for the parameters or add a check to https://github.com/indielab/autoscout24/pull/1/commits/6afd5c2f01cbe70040e37d84c2bfeaad642dbc5b#diff-40d7473316ef0bb3ba4c4d937b16ae3d1cc47e0d65815034ee2b5be6a74cb11eR25 . I will take a deeper look at the api´s responses and add some kind of check if thats fine for you

Reply:
I think missing or wrong configured constructor data is very annoying for developers. Debugging the data to find out a constructor param was missing is not a good experience. I will add the empty check and merge afterwards :+1: